0000-interledger-protocol-4/0000-interledger-protocol-4.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Interledger Protocol V4 (ILPv4)
+draft: 1
+---
+
+# Interledger Protocol V4
+
+Interledger is a protocol for sending packets of money across different payment networks or ledgers. ILPv4 is a simplification of previous versions of the protocol designed primarily for "penny switching", or routing large volumes of smaller-value packets. ILPv4 can be integrated with any type of ledger, including those not built for interoperability, and it can be used with a variety of higher-level protocols that implement features ranging from quoting to sending larger amounts of value with chunked payments. 
+
+## Overview
+
+### Design Goals
+
+- **Neutrality** - Interledger is not tied to any company, currency, or network.
+- **Interoperability** - ILP should be usable across any type of ledger, even those that were not built for interoperability.
+- **Security** - Senders, connectors, and receivers should be protected from one another and especially isolated from risks posed by parties that they are indirectly connected to. Connectors should not be able to steal money from senders, and senders should not be able to tie up too much of connectors' funds or otherwise interfere with their operation.
+- **Simplicity** - The core ILP should be as simple as possible. There is room for significant variability at lower and higher layers of the [Interledger stack](../0001-interledger-architecture/0001-interledger-architecture.md#interledger-protocol-suite), but the Interledger layer is the one part where widespread agreement is needed for interoperability.
+- **End-to-End Principle** - Inspired by the Internet, any features that do not need to be implemented by the core Interledger Protocol and network of connectors should be built into the edges of the network (the sender and receiver).
+
+### Terminology
+
+- **Balance** - Connectors may track the credit each customer or peer holds with them. Accounts may be pre-funded (in which case the customer or peer trusts the connector for the value of the account balance) or post-funded (in which case the connector trusts the customer or peer for the amount the account is allowed to go negative). Connectors may refuse to process further ILP packets if an account balance goes too low.
```

Comment:
How about: "Because this balance represents un-settled debt between direct neighbors, either party can default on it. If an account is pre-funded, the customer/peer trusts the connector not to zero their balance. If an account is post-funded, the connector trusts the customer/peer to later settle this debt. Either party may stop transacting if the balance exceeds their trust in the other party. The amount of trust required to practically transact is related inversely to the speed of the ledger used for settlement."